src/star.css

```diff
@@ -0,0 +1,25 @@
+.stars--4 {
+  height: 16px;
+  display: flex;
+}
```

Comment:
These styles are redundant, delete them

Reply:
i can delete only "height", if i`ll delete "display" stars are breaking

Reply:
@vallerii so move this property to some other selector. If I'll change modifier to stars--3, everything will be broken